warehouse/email.py

```diff
@@ -19,15 +19,18 @@
 
 
 @tasks.task(bind=True, ignore_result=True, acks_late=True)
-def send_email(task, request, body, recipients, subject):
+def send_email(task, request, body, recipients, subject, send_to_bcc=False):
```

Comment:
I think I would prefer if this more closely mimicked the arguments that `Message` takes, so something like:

```python
def send_email(task, request, body, subject, recipients=None, bcc=None)
```
(FYI, changing the order here might require updates elsewhere where we're passing this positional arguments)

And then since the default arguments for `Message` are already `None`, we can call it like:

```python
message = Message(
    body=body,
    recipients=recipients,
    bcc=bcc,
    sender=...,
    subject=subject,
)

Reply:
Ok it makes sense to do it that way. Thanks.
That'll be quite a big change, since this is being called everywhere as positional argument. 
I will prepare a separate PR just to update `recipients` into keyword only argument in `send_email` method.